sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -1054,66 +1054,71 @@ def task_worker(self):
             namespace = redisSelectObj.getDbConnector().getNamespace()
             asic_index = multi_asic.get_asic_index_from_namespace(namespace)
 
-            (port, op, fvp) = status_tbl[asic_index].pop()
-            if fvp:
-                # This check might be redundant, to check, the presence of this Port in keys
-                # in logical_port_list but keep for now for coherency
-                # also skip checking in logical_port_list inside sfp_util
-                if port not in y_cable_tbl_keys[asic_index]:
-                    continue
-
-                fvp_dict = dict(fvp)
-
-                if "state" in fvp_dict:
-                    # got a state change
-                    new_status = fvp_dict["state"]
-                    (status, fvs) = y_cable_tbl[asic_index].get(port)
-                    if status is False:
-                        helper_logger.log_warning("Could not retreive fieldvalue pairs for {}, inside state_db table {}".format(
-                            port, y_cable_tbl[asic_index]))
+            while True:
+                (port, op, fvp) = status_tbl[asic_index].pop()
```

Comment:
i always think swss-common should provide pops that allows user to pop everything. @qiluo-msft, we really should fix that.

Reply:
Yes, I will fix it.